package.json

```diff
@@ -7,11 +7,11 @@
     "clean": "lerna run clean",
     "functional-test": "lerna run build && nyc mocha \"libraries/functional-tests/tests/*.test.js\"",
     "browser-functional-test": "cd libraries/browser-functional-tests && node nightwatch.js -e",
-    "test": "lerna run build && nyc mocha \"libraries/@(adaptive*|bot*)/tests/**/*.test.js\" --exit",
+    "test": "lerna run build && nyc mocha \"libraries/@(adaptive*|bot*)/tests/**/*.test.js\" --exit --check-leaks",
```

Comment:
I think it would be good to have the `--check-leaks` flag in both `test` and `test-coverage` to prevent this in the future, but I'm willing to remove them if reviewers disagree. If a leak is detected, it makes the test fail. As far as I can tell, leaks are detected deterministically.

Reply:
I think this is worth trying out. If it causes pain to the dev team we can evaluate an alternative path for keeping track of test leaks.